temporal/api/cloud/cloudservice/v1/service.proto

```diff
@@ -128,6 +128,13 @@ service CloudService {
         };
     }
 
+    // Remove a region from a namespace
+    rpc RemoveNamespaceRegion (RemoveNamespaceRegionRequest) returns (RemoveNamespaceRegionResponse) {
+        option (google.api.http) = {
+            delete: "/cloud/namespaces/{namespace}/{region}",
```

Comment:
```suggestion
            delete: "/cloud/namespaces/{namespace}/regions/{region}",
```

What if my region was named `add-region`? We have not properly reserved all children of `/cloud/namespaces/{namespace}` as regions, so it's not like `POST /cloud/namespaces/{namespace}` adds a region. Also, `POST /cloud/namespaces/{namespace}/add-region` is probably not the best HTTP API for adding, I'd consider adding an additional binding (or even making the BC change to remove the other) to `POST /cloud/namespaces/{namespace}/regions`.

Reply:
all make sense